app/policies/behaviour_policy.rb

```diff
@@ -0,0 +1,32 @@
+class BehaviourPolicy < ApplicationPolicy
+  def update?
+    is_bot_owner? or is_collaborator?
+  end
+
+  def destroy?
+    !is_front_desk? and (is_bot_owner? or is_collaborator?)
+  end
+
+  def is_front_desk?
+    record.kind == 'front_desk'
+  end
+
+  def is_bot_owner?
```

Comment:
We could define a module/concern that requires when included to define a `def bot` and that would provide a `def is_collaborator_or_owner?` or `def at_least_collaborator?` and reduce the boilerplate in the policies. WDYT?